src/core/public/chrome/ui/header/header_controls_container.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
```

Comment:
for my own knowledge, how do we decide which headers we put on top of new files?

Reply:
Discussed offline.